cmd/ww/server.go

```diff
@@ -338,13 +373,22 @@ func server(args ...string) {
 		Prompt:     autocert.AcceptTOS,
 		HostPolicy: autocert.HostWhitelist(strings.Split(*whitelist, ",")...),
 	}
+
+	if customCertFlag != "" {
```

Comment:
let's check both cert and key are set. if one isn't print an error message and exit.